config/shared_config_test.go

```diff
@@ -264,64 +258,64 @@ func TestLoadSharedConfigFromFile(t *testing.T) {
 			Expected: SharedConfig{Region: "default_region"},
 		},
 		1: {
-			Profile:  "alt_profile_name",
+			Profile:  "profile alt_profile_name",
 			Expected: SharedConfig{Region: "alt_profile_name_region"},
 		},
 		2: {
-			Profile:  "short_profile_name_first",
-			Expected: SharedConfig{Region: "short_profile_name_first_short"},
+			Profile:  "profile short_profile_name_first",
+			Expected: SharedConfig{Region: "short_profile_name_first_alt"},
 		},
 		3: {
-			Profile:  "partial_creds",
+			Profile:  "profile partial_creds",
 			Expected: SharedConfig{},
 		},
 		4: {
-			Profile: "complete_creds",
+			Profile: "profile complete_creds",
 			Expected: SharedConfig{
 				Credentials: aws.Credentials{
 					AccessKeyID:     "complete_creds_akid",
 					SecretAccessKey: "complete_creds_secret",
-					Source:          fmt.Sprintf("SharedConfigCredentials: %s", testConfigFilename),
+					Source:          fmt.Sprintf("SharedConfigCredentials: %s", ""),
 				},
 			},
 		},
 		5: {
-			Profile: "complete_creds_with_token",
+			Profile: "profile complete_creds_with_token",
 			Expected: SharedConfig{
 				Credentials: aws.Credentials{
 					AccessKeyID:     "complete_creds_with_token_akid",
 					SecretAccessKey: "complete_creds_with_token_secret",
 					SessionToken:    "complete_creds_with_token_token",
-					Source:          fmt.Sprintf("SharedConfigCredentials: %s", testConfigFilename),
+					Source:          fmt.Sprintf("SharedConfigCredentials: %s", ""),
 				},
 			},
 		},
 		6: {
-			Profile: "full_profile",
+			Profile: "profile full_profile",
 			Expected: SharedConfig{
 				Credentials: aws.Credentials{
 					AccessKeyID:     "full_profile_akid",
 					SecretAccessKey: "full_profile_secret",
-					Source:          fmt.Sprintf("SharedConfigCredentials: %s", testConfigFilename),
+					Source:          fmt.Sprintf("SharedConfigCredentials: %s", ""),
```

Comment:
We don't have data for source name when we loadConfigFromSection.  But this test is not really needed.